tools/outline_proxy_controller/dist/install_linux_service.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2018 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+readonly PREFIX=/usr/local
+readonly SERVICE_NAME=outline_proxy_controller.service
+
+# Copy/update the service's files.
```

Comment:
Looking at the diff with `src/electron/install_linux_service.sh`, I have some questions:
* Is this equivalent to deleting the files and re-installing them? 
* Are we not configuring the daemon to run on boot anymore?